CHANGELOG.md

```diff
@@ -40,6 +40,9 @@
   the page before recording the observation, set the new `if_page_missing`
   option to `HistoryMetadataPageMissingBehavior::InsertPage`.
 
+### Remote Settings
+- Changed the name of the `Attachment` struct (used inside `RemoteSettingsRecord`) to `AttachmentMetadata` to better reflect the purpose.
```

Comment:
Checked against `firefox-android` and it breaks. Don't exactly know how to notify the teams (slacked them earlier).